### ISSUE TITLE
Bottom of tray Z value seems to be non existent 

### ISSUE THREAD

**Issue by CARDBOARDCAR** [node-dojo/dojo-recursive-bins#1]
![screen shot in blender](https://github.com/node-dojo/dojo-recursive-bins/assets/158329374/68a0971f-1a55-4dff-aefb-5e3853c28ce7)

There there doesn't appear to be a z value for the bottom of the trays when exported out to a slicing software. Even when in blender if you bring your zoom into the bottom of the tray as seen in image you'll see its super thin. Can you make this an adjustable value or tie it together with the wall 
![screen shot bbs](https://github.com/node-dojo/dojo-recursive-bins/assets/158329374/ae51d679-fb48-4cda-bfb1-b79b8333a634)
thickness.

**Comment by veilofsecurity** [comment]
Seeing the same thing. Get an invalid model when exporting to stl and importing to cura. Looks like the build plate exports with it and walls and scale aren't quite right.
<img width="1148" alt="image" src="https://github.com/node-dojo/dojo-recursive-bins/assets/40548677/ad3e0a9d-2c7e-4ec5-af42-ec84b02089bf">

**Comment by node-dojo** [comment]
![Screen Recording 2024-01-31 at 2 23 22 PM](https://github.com/node-dojo/dojo-recursive-bins/assets/157924548/07759513-e640-45c6-9e46-cb070ff3be97)

bottom of the tray should now be exporting at the same thicknesses of as the bin walls

**Comment by veilofsecurity** [comment]
Hmm still seeing an error but this is the first time I have worked with Blender so maybe it's me. I open the file in Blender, adjust a couple parameters like wall thickness, click make exportable. Then I used File->Export->STL, leave defaults. Open the STL in Cura 5.6 (tried 5.4 at first) and still get this:
<img width="535" alt="image" src="https://github.com/node-dojo/dojo-recursive-bins/assets/40548677/5d4f6280-a830-44b3-8c13-d198c91fb563">

**Comment by veilofsecurity** [comment]
Not sure if this was the correct way to do this but after clicking make exportable I just unlinked the ankermake build plate under scene collection, set print layers to 500 and re-exported. This got rid of the build plate from the stl. I then scaled up the model in CURA and it does slice now but doesn't seem to have a bottom, just walls.
<img width="1077" alt="image" src="https://github.com/node-dojo/dojo-recursive-bins/assets/40548677/c792f9f1-52f9-406d-a492-da3e00966f06">

**Comment by node-dojo** [comment]
sorry Ive only patched this bug on the 2-step version. you mind trying that while i work on the patch for the red bin version?

**Comment by node-dojo** [comment]
should be resolved in both versions now

**Comment by CARDBOARDCAR** [comment]
It works now. Thanks for fixing that! 

Any chance you'll, at some point, add in a place to see the dimensions of each bin. Perhaps have them displayed above the bin selected prior to export  so a person can confirm if a bin that size will fit items planned for that location.

**Comment by veilofsecurity** [comment]
Oh, very cool. The build plate thing was a me issue. I just checked the selection only box on export and that is fixed. I still think there is a bit of a scaling issue on export though. On export I set the scale to 100 and the output doesn't seem to match. Here is a simple example: I set easy parameters to get a nice 4x4 grid so it should be roughly 123.75x123.75x100 (500-5/4) tray. Instead in cura I see 118x110x40. For the LxW I thought maybe your X and Y were outside the drawer instead of inside but not sure why Z would be less than half the height.
<img width="1917" alt="image" src="https://github.com/node-dojo/dojo-recursive-bins/assets/40548677/d6ec95c7-5088-43e5-a707-a0b86ba814c6">
<img width="1364" alt="image" src="https://github.com/node-dojo/dojo-recursive-bins/assets/40548677/69eb825a-5073-4296-8cea-27aea75d127c">

And thanks for working on this btw. This is a super awesome project. I've been thinking over manually building something like this for a drawer for quite a while but never got around to drawing it out and then stumbling through Fusion360.
